docs/docs/getting-started.rst

```diff
@@ -0,0 +1,75 @@
+.. _getting-started:
+
+Getting started
+===============
+
+Where to install Dru?
+---------------------
+
+Dru can be thought as of an engine that provides its clients with answers on blockchain-related queries. As such, it can be run on your local computer, but bear in mind that if the blockchain of a chosen cryptocurrency is large, you need to be equipped with enough hardware (especially RAM). Apart from that, some of the queries can be computationally complex, so this is another factor to consider. This is why it is suggested to have Dru installed on either a server or a workstation that is always on and has enough resources. This will create a typical client-server architecture and many researchers will have the opportunity to use Dru at the same time and gain access to already computed queries' results. Nevertheless, this is not a necessity and you can install Dru on your local computer.
+
+Installing Dru
+--------------
+
+Installing docker
+~~~~~~~~~~~~~~~~~
+
+Dru runs in the Docker environment. Before continuing with installing Dru, please make sure that you have Docker CE installed. The following guide shows how to install Docker CE: https://docs.docker.com/install/
+
+Cloning the repository
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you have Docker CE installed, you can proceed to cloning the Dru repository::
+
+    git clone https://github.com/bergplace/Dru.git
+
+Note, that the Docker compose file provided by Dru has version 3.3 and as such it requires Docker in version 17.06.0 or above. For more details on compatibility, please visit this webpage: https://docs.docker.com/v17.09/compose/compose-file/
+
+Creating the configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, use the default configuration file as a template for your Dru instance configuration::
+
+    cd Dru
+    cp dru.conf.default dru.conf
```

Comment:
you should add that "default configuration should be perfectly suitable for running Dru locally"

Reply:
and in that case you dont even need to copy the conf yourself, if you run locally and have docker and docker compose, just run make start and it will work